helm-chart-sources/pulsar/templates/proxy/proxy-configmap.yaml

```diff
@@ -122,3 +122,15 @@ data:
   PULSAR_MEM: {{ .Values.proxy.configData.PULSAR_MEM }}
   PULSAR_GC: {{ .Values.proxy.configData.PULSAR_GC }}
 {{- end }}
+{{- if .Values.proxy.kafkaOnPulsarEnabled }}
+  # enabling Kafka-on-Pulsar
+  proxyProtocolHandlerDirectory: "./proxyprotocols"
```

Comment:
Should the directory be defined here ? If we have multiple proxy protocols (eg. RabbitMQ), it will conflict

Reply:
you have one "protocols" directory for Broker PHs and proxyprotocols for Proxy PHs,
no conflict
see the PIP-92

Reply:
Yes but in this PR, it's under `kafkaOnPulsarEnabled`. I think it should be defined outside of this condition or it will collide with the one under the future `rabbitMQEnabled`.

Reply:
I don't know how to handle this with Helm
any suggestion is appreciated

Reply:
@eolivelli as I understood from https://helm.sh/docs/chart_template_guide/functions_and_pipelines/ and https://helm.sh/docs/chart_template_guide/function_list/#or
it should look like 
```
{{- if or .Values.proxy.kafkaOnPulsarEnabled .Values.proxy.rabbitMQEnabled }}
```

Reply:
My main question: do we need specific sections for each protocol handler? Or can we get away with a single generic proxy protocol handler section?

Reply:
I think proxyProtocolHandlerDirectory could be set globally outside of any condition. As long as you don't set proxyMessagingProtocols it isn't used anyway.

Reply:
> I think proxyProtocolHandlerDirectory could be set globally outside of any condition. As long as you don't set proxyMessagingProtocols it isn't used anyway.

I don't feel strongly either way. It is generic now, though, which addresses the original comment.